### PR TITLE
Re-render feedstock, upgrade version to 1.13.1

### PR DESCRIPTION
**Pull request by kyleabeauchamp**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

Wanted to get a recent version of tensorflow built.  Once we clean up the recipe and build process here, then we can start planning for the 1.13 release that is upcoming.  

FYI I think the tensorflow CPU wheel files use SSE4.2 instructions (and possibly AVX?), so this may push the boundaries of our supported CPU architectures.

### CODE REVIEW

**Comment by kyleabeauchamp** [comment]
PS: we expect the py37 build to fail because there is no such build for tf 1.12.  However, I wanted to start getting the build architecture in place.

**Comment by kyleabeauchamp** [comment]
It looks like keras_applications is now a pre-req, so I'll have to add conda-forge recipes for that.

**Comment by kyleabeauchamp** [comment]
https://github.com/conda-forge/staged-recipes/pull/7828/files

**Comment by jjhelmus** [comment]
[keras-applications](https://github.com/conda-forge/keras-applications-feedstock) is the name of the package on conda-forge.

**Comment by kyleabeauchamp** [comment]
doh, fixed ;)

**Comment by kyleabeauchamp** [comment]
Hmm, looks like there's an issue where tensorflow fails upon import due to old versions of numpy (from a775aca build), but something is pinning to older versions of numpy.

**Comment by kyleabeauchamp** [comment]
PS: here is a link to the tf/numpy import issue that I found; issue suggested upgrading version: https://github.com/tensorflow/tensorflow/issues/25942

**Comment by njzjz** [comment]
[1.13.0 has released](https://github.com/tensorflow/tensorflow/releases).

**Comment by kyleabeauchamp** [comment]
I've updated this PR to use the latest 1.13.1 release.  I had to disable windows builds to get this to work, but it's now passing all but one of the other platform tests.  The remaining Travis CI failure appears to be a timeout (not 100% sure).  What do folks think?

**Comment by mariusvniekerk** [comment]
Yeah given that the osx ones on azure complete (but take 200 minutes) its probably a timeout

**Comment by mariusvniekerk** [comment]
in order for this to release binaries we'd need to switch the default publisher

**Comment by kyleabeauchamp** [comment]
Can you explain that a bit more, never heard of "default publisher".

File: recipe/meta.yaml [hunk 1]
```diff
@@ -30,15 +31,17 @@ requirements:
     - absl-py >=0.1.6
     - astor >=0.6.0
     - gast >=0.2.0
-    - numpy >=1.13.3
+    - numpy >=1.16.1
```
Comment (jjhelmus) [inline recipe/meta.yaml]: The wheel indicates compatibility with nuimpy >=1.13.3

File: recipe/meta.yaml [hunk 2]
```diff
@@ -30,15 +31,17 @@ requirements:
...
     - six >=1.10.0
     - protobuf >=3.6.0
```
Comment (jjhelmus) [inline recipe/meta.yaml]: protobuf >=3.6.1

File: recipe/meta.yaml [hunk 3]
```diff
@@ -30,15 +31,17 @@ requirements:
...
-    - tensorboard 1.10.*
+    #- tensorboard 1.12.*  # NB: tensorboard is now bundled in the tensorflow wheel.
     - termcolor >=1.1.0
     - grpcio >=1.8.6
     - enum34 >=1.1.6              # [py<34]
     - backports.weakref >=1.0rc1  # [py<34]
     - mock >=2.0.0                # [py2k]
+    - keras-applications
```
Comment (jjhelmus) [inline recipe/meta.yaml]: has a lower bound of 1.0.6 in the wheel `keras-application >=1.0.6`

File: recipe/meta.yaml [hunk 4]
```diff
@@ -30,15 +31,17 @@ requirements:
...
+    - keras-preprocessing
```
Comment (jjhelmus) [inline recipe/meta.yaml]: Where wheel specifies this as `keras-preprocessing >=1.0.5`

File: recipe/meta.yaml [hunk 5]
```diff
@@ -30,15 +31,17 @@ requirements:
     - absl-py >=0.1.6
     - astor >=0.6.0
     - gast >=0.2.0
-    - numpy >=1.13.3
+    - numpy >=1.16.1
     - six >=1.10.0
     - protobuf >=3.6.0
-    - tensorboard 1.10.*
+    #- tensorboard 1.12.*  # NB: tensorboard is now bundled in the tensorflow wheel.
```
Comment (jjhelmus) [inline recipe/meta.yaml]: tensorboard is a separate package on PyPI and in conda.  The tensorflow wheel does include the tensorboard entry point but tensorboard is still a dependency.

**Comment by jjhelmus** [comment]
Tensorflow 1.13.1. added a new run dependency of `tensorflow-estimator >=1.13.0,<1.14.0rc0`

File: recipe/meta.yaml (hunk 1 above)
Comment (kyleabeauchamp) [reply recipe/meta.yaml]: I was unable to get the import to work due to an issue similar to https://github.com/tensorflow/tensorflow/issues/559, which was resolved by a numpy upgrade.

**Comment by kyleabeauchamp** [comment]
Made some of the requested changes.

**Comment by kyleabeauchamp** [comment]
Just need to build some of the other deps: 

```
conda_build.exceptions.DependencyNeedsBuildingError: Unsatisfiable dependencies for platform linux-64: {'tensorboard=1.13', "tensorflow-estimator[version='>=1.13.0,<1.14.0rc0']"}

```

**Comment by dhirschfeld** [comment]
> *Tensorflow 1.13.1. added a new run dependency of `tensorflow-estimator >=1.13.0,<1.14.0rc0`*

Isn't that a terrible version pinning? IIUC that will match all alpha and beta releases of 1.14.0 which surely can't be desired?

Even though that version is specified in the [tensorflow repo](https://github.com/tensorflow/tensorflow/blob/23f78239169810a542dfa853fa524460070db004/tensorflow/tools/pip_package/setup.py#L50-L63) it would probably pay to fix it to `1.14.0a0` to prevent future breakage or even just `1.13.*`

**Comment by kyleabeauchamp** [comment]
I would be fine with that change, but I'll let @jjhelmus be the deciding vote.

**Comment by kyleabeauchamp** [comment]
I think we need someone to merge the auto-generated tensorboard bump PR: https://github.com/conda-forge/tensorboard-feedstock/pull/19

**Comment by kyleabeauchamp** [comment]
OK the tensorboard dependency is now uploaded, next we need to build the tensorflow-estimator recipe.

**Comment by kyleabeauchamp** [comment]
OK, made PR for estimator: https://github.com/conda-forge/staged-recipes/pull/7895

**Comment by henryiii** [comment]
Any progress?

**Comment by marcelotrevisani** [comment]
@conda-forge-admin, please rerender

**Comment by kyleabeauchamp** [comment]
This build looks green?  Any more reviewers?

**Comment by dhirschfeld** [comment]
LGTM but as @jjhelmus is the assigned reviewer I was going to leave merging to him...

**Comment by jjhelmus** [comment]
LGTM, thanks for all your work on this @kyleabeauchamp, sorry it too so long for a review.

**Pull request merged**